routes/api/User/index.js

```diff
@@ -0,0 +1,32 @@
+const router = require('express').Router();
+const authenticateJWT = require ('../../../middleware/authenticateJWT')
+var UserController = require('../../../controllers')
+const multer = require('multer')
+
+const storage = multer.diskStorage({
+    destination: function(req,file,cb) {
+        cb(null, './uploads')
+    },
+    filename:function(req,file,cb) {
+        cb(null, Date.now().toString() + file.originalname )
+    }
+})
```

Comment:
Multer and its related services should be inside the services folder not inside the routes folder 
The routes folder should only contain routes and related http methods